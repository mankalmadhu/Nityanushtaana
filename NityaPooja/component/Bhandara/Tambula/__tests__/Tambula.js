import 'react-native';
import React from 'react';
import ತಾ೦ಬೂಲ from '../Tambula';

import Renderer from 'react-test-renderer';

jest.mock('../../../Avarana/Avarana',()=>'ಆವರಣ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');
jest.mock('../../../Vishaya/Purvanga',()=>'ಪೂರ್ವೋತ್ತರ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');
jest.mock('../../../Vishaya/Vidhana',()=>'ವಿಧಾನ');

test("ತಾ೦ಬೂಲ" , () => {

expect(Renderer.create(
    <ತಾ೦ಬೂಲ />
).toJSON()).toMatchSnapshot();

});