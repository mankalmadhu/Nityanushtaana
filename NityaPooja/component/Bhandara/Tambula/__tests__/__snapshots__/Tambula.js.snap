// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ತಾ೦ಬೂಲ 1`] = `
<ಆವರಣ>
  <ಶೀರ್ಷಿಕೆ>
    ತಾ೦ಬೂಲ
  </ಶೀರ್ಷಿಕೆ>
  <ವಿನಿಯೋಗ>
    ನಾಭ್ಯಾ॑ ಆಸೀದ॒೦ತರಿ॑ಕ್ಷಮ್ । ಶೀ॒ರ್ಷ್ಣೋ  ದ್ಯೌಸ್ಸಮ॑ವರ್ತತ । ಪ॒ದ್ಭ್ಯಾಂ ಭೂಮಿ॒ರ್ದಿಶ॒ಶ್ಯೋತ್ರಾ᳚ತ್ । ತಥಾ॑ ಲೋ॒ಕಾꣳ ಅ॑ಕಲ್ಪಯನ್ ।
  </ವಿನಿಯೋಗ>
  <ಪೂರ್ವೋತ್ತರ>
    ಶ್ರೀ ಇಷ್ಟ ದೇವತಾಭ್ಯೋ ನಮಃ । ಪೂಗಿಫಲ ತಾ೦ಬೂಲ೦ ನಿವೇದಯಾಮಿ ॥
  </ಪೂರ್ವೋತ್ತರ>
</ಆವರಣ>
`;
